src/main/java/org/dinosaur/foodbowl/domain/photo/ImageUtils.java

```diff
@@ -0,0 +1,62 @@
+package org.dinosaur.foodbowl.domain.photo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.multipart.MultipartFile;
+
+@Component
+public abstract class ImageUtils {
+
+    @Value("${file.dir}")
+    protected String fileDir = "foodbowl-res";
```

Comment:
`res`는 무슨 의미인지 알 수 있을까요?

Reply:
`@Value`를 사용하는만큼 디폴트 값은 지정하지 않는게 어떨까요?

Reply:
> `res`는 무슨 의미인지 알 수 있을까요?
`res`는 `resources` 의미로 사용하였습니다!

Reply:
> `@Value`를 사용하는만큼 디폴트 값은 지정하지 않는게 어떨까요?

Spring 을 사용하지 않고 테스트하고 싶어서 디폴트 값을 지정했는데
혹시 추천하시는 다른 방법이 있을까요

Reply:
```suggestion
public ImageUtils(@Value("${file.dir}") String fileDir) {
    this.fileDir = fileDir;
}
```
요렇게하면 생성자로도 값을 넣어줄 수 있습니다!

Reply:
`bucket/photo`, `bucket/thumbanil`
```
ㄴ src
ㄴ bucket
    ㄴ photo
    ㄴ thumbnail
```